onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -251,6 +251,10 @@ def bind_cpu_input(self, name, arr_on_cpu):
         :param name: input name
         :param arr_on_cpu: input values as a python array on CPU
         '''
+        # Hold a reference to the numpy object as the bound OrtValue is backed 
+        # directly by the data buffer of the numpy object and so the numpy object
+        # must be around until this IOBinding instance is around
+        self._numpy_obj_references.append(arr_on_cpu)
```

Comment:
hold onto a reference of the numpy object as the underlying OrtValue uses its buffer directly and it should be tied to the life  of the IOBinding instance

Reply:
Is this fix ready to apply?